### DISCUSSION TITLE
IP Spoofing

### DISCUSSION THREAD

**Discussion by BigboiKilljoy** [YimMenu/YimMenu#1466] [General]
I got DDoS by Username: [REDACTED], Rockstar ID: [REDACTED]. All would have be fine if i remembered to turn on my VPN before joining a sensitive bitch who tried to crash me for no reason but i didn't. is there anyway i can be saved from this type of bullshit by you cool hacker people and no VPN

**Comment by Yimura** [comment]
Sorry but this is not a doxing group, please don’t post any information of other people’s accounts.

**Reply by BigboiKilljoy** [reply]
I dont know how you consider that doxing in anyway, i mean id gladly put my own username and Rockstar ID right here for everyone to see just to show you how dumb i think this is but i bet you would redact my own info too. Can you just tell me how i can do something about this person.

**Reply by Yimura** [reply]
Stop caring and enable RID join protection.

**Reply by BigboiKilljoy** [reply]
But i have friends and if i turn that option on they wont be able to join me. I hate sensitive people. they're the worst and deserve to be broken. I understand that you wont help me though so goodbye for now

**Reply by Yimura** [reply]
Disable it when they want to join or you join them instead?

**Comment by ghost** [comment]
I've been stalked for months through this game. Not much you can really do (that hasn't already been said) other than not play or use a firewall designed for gta online, like securosurf or guardian, or join a crew like the safe crew (who are great, by the way, if you like to actually play the game). 

The only way to win is to not engage with these creeps. That's what they want.
